jms/src/main/java/com/sun/ts/tests/jms/ee/mdb/mdb_msgHdrT/mdb_msgHdrT_ejb.jar.sun-ejb-jar.xml

```diff
@@ -19,7 +19,7 @@
 -->
 
 <sun-ejb-jar>
-	<disable-nonportable-jndi-names>true</disable-nonportable-jndi-names>
+	<!-- <disable-nonportable-jndi-names>true</disable-nonportable-jndi-names> -->
```

Comment:
I had to comment this out to avoid NameNotFoundException.

Reply:
Maybe just remove it instead of commenting out?